src/brevitas/export/onnx/standard/qcdq/handler.py

```diff
@@ -75,6 +78,33 @@ def quantize_fn(self, x, scale, zero_point, dtype, axis):
         return QuantizeLinearFn.apply(x, scale, zero_point, dtype, axis)
 
 
+class StdDynamicQDQCastONNXMixin(DynamicQMixin, StdDQCastONNXMixin, ABC):
```

Comment:
This should inherit from `StdQCDQCastONNXMixin` and override just `validate` and `quantize_fn`